### PR TITLE
Apply style to vertical features section

### PR DESCRIPTION
**Pull request by Yukit02**

### CODE REVIEW

File: src/feature/FeatureImageGrid.tsx [hunk 1]
```diff
@@ -1,36 +1,39 @@
+/* eslint-disable jsx-a11y/alt-text */
 import React, { ReactNode } from 'react';
 
 type IFeatureImageGridProps = {
   title: ReactNode;
   children: ReactNode;
   imageSrc: string;
+  imageSrcTight: string;
```
Comment (hoonsubin) [inline src/feature/FeatureImageGrid.tsx:8]: please don't add two image sources

**Review by hoonsubin — CHANGES_REQUESTED** [review]

File: src/feature/FeatureImageGrid.tsx (hunk 1 above)
Comment (Yukit02) [reply src/feature/FeatureImageGrid.tsx]: Is there a problem with this fix?

File: src/feature/FeatureImageGrid.tsx [hunk 2]
```diff
@@ -10,27 +11,28 @@ type IFeatureImageGridProps = {
...
+      props.imageOnRight ? 'md:flex-row-reverse' : ''
+    } font-bold font-robot not-sr-only`}
+  >
+    <div>
+      <img
+        src={`${props.imageSrc}.svg`}
+        className={`${props.imageOnRight ? 'order-last' : ''} md:hidden xl:block w-full`}
+      />
+      <img src={`${props.imageSrc}-tight.svg`} className="hidden md:block xl:hidden w-full" />
+    </div>
```
Comment (Yukit02) [inline src/feature/FeatureImageGrid.tsx:25]: Is there a problem with this fix?

File: src/feature/FeatureImageGrid.tsx [hunk 3]
```diff
@@ -10,27 +11,28 @@ type IFeatureImageGridProps = {
...
-  <div className="grid grid-cols-1 grid-rows-2 md:grid-cols-2 md:grid-rows-1 justify-items-stretch">
-    <div className={`section-art ${props.imageOnRight ? 'order-last' : ''}`} />
+  <div
+    className={`flex flex-col md:flex-row 2xl:max-w-screen-2xl mx-auto ${
+      props.imageOnRight ? 'md:flex-row-reverse' : ''
+    } font-bold font-robot not-sr-only`}
+  >
+    <div>
+      <img
+        src={`${props.imageSrc}.svg`}
```
Comment (hoonsubin) [inline src/feature/FeatureImageGrid.tsx:21]: the property argument value should contain the file extension. manually appending `.svg` is not good

File: src/feature/FeatureImageGrid.tsx [hunk 4]
```diff
@@ -10,27 +11,28 @@ type IFeatureImageGridProps = {
...
+        className={`${props.imageOnRight ? 'order-last' : ''} md:hidden xl:block w-full`}
+      />
+      <img src={`${props.imageSrc}-tight.svg`} className="hidden md:block xl:hidden w-full" />
```
Comment (hoonsubin) [inline src/feature/FeatureImageGrid.tsx:24]: There is not error handling for images that do not end with `-tight.svg`

**Review by hoonsubin — CHANGES_REQUESTED** [review]

File: src/feature/FeatureImageGrid.tsx (hunk 4 above)
Comment (Yukit02) [reply src/feature/FeatureImageGrid.tsx:24]: Sry I don't know how to fix this.
plz tell me some idea.

Comment (hoonsubin) [reply src/feature/FeatureImageGrid.tsx:24]: Remove the hard coded `.svg`

Comment (Yukit02) [reply src/feature/FeatureImageGrid.tsx:24]: I thought Can not display images when `.svg` removed
We need to toggle with screen size for image.

Comment (hoonsubin) [reply src/feature/FeatureImageGrid.tsx:24]: You can still use a vector file as a source (pass the full file path as the param). I believe I mentioned about making the svg file as a component (e.g., TwitterIcon.tsx) and change the style from there.
Plus, don't duplicate the same element. Use one element and resize it depending on the break point

**Pull request merged**